src/sentry/testutils/cases.py

```diff
@@ -243,6 +249,12 @@ def login_as(
             # XXX(dcramer): we're calling the internal method to avoid logging
             request.superuser._set_logged_out()
         elif request.user.is_superuser and superuser:
+            request._body = json.dumps(
```

Comment:
I would consider adding a setting to toggle checking SU ACCESS reason, as it would be useful not to have to provide this mocked data, and additionally in development it is probably worth hiding for better dx anyways.